api/validators/MerchStoreRequests.ts

```diff
@@ -308,3 +309,9 @@ export class EditOrderPickupEventRequest implements IEditOrderPickupEventRequest
   @IsDefined()
   pickupEvent: OrderPickupEventEdit;
 }
+
+export class GetCartRequest implements IGetCartRequest {
+  @IsDefined()
+  @IsNotEmpty()
+  items:string[];
+}
```

Comment:
```suggestion
  @IsNotEmpty()
  items: string[];
}
```

Reply:
@shravanhariharan2 can we update the linter to include this?